lib/stats/DefaultStatsPrinterPlugin.js

```diff
@@ -1348,45 +1348,15 @@ const AVAILABLE_FORMATS = {
 		}
 		return `${boldQuantity ? bold(time) : time}${unit}`;
 	},
-	formatError: (message, { green, yellow, red }) => {
-		if (message.includes("\u001B[")) return message;
-		const highlights = [
-			{ regExp: /(Did you mean .+)/g, format: green },
-			{
-				regExp: /(Set 'mode' option to 'development' or 'production')/g,
-				format: green
-			},
-			{ regExp: /(\(module has no exports\))/g, format: red },
-			{ regExp: /\(possible exports: (.+)\)/g, format: green },
-			{ regExp: /(?:^|\n)(.* doesn't exist)/g, format: red },
-			{ regExp: /('\w+' option has not been set)/g, format: red },
-			{
-				regExp: /(Emitted value instead of an instance of Error)/g,
-				format: yellow
-			},
-			{ regExp: /(Used? .+ instead)/gi, format: yellow },
-			{ regExp: /\b(deprecated|must|required)\b/g, format: yellow },
-			{
-				regExp: /\b(BREAKING CHANGE)\b/gi,
-				format: red
-			},
-			{
-				regExp:
-					/\b(error|failed|unexpected|invalid|not found|not supported|not available|not possible|not implemented|doesn't support|conflict|conflicting|not existing|duplicate)\b/gi,
-				format: red
-			}
-		];
-		for (const { regExp, format } of highlights) {
-			message = message.replace(
-				regExp,
-				/**
-				 * @param {string} match match
-				 * @param {string} content content
-				 * @returns {string} result
-				 */
-				(match, content) => match.replace(content, format(content))
-			);
```

Comment:
Why you changed it? Please don't touch code if you don't need it or provide a description about why you changed it and what problems did you solve when you changed it